templates/views/index.html

```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
         <div class="row">
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
                 <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                    <p class="__shecodetestimp">This Sca mentoring program was like a breathe of fresh air that I needed
+                        since December 2019. I'm so filled as I joined
+                        and has been growing on my learning path (Php/laravel).
+                        In fact my mentor is the best that anyone can have. She's not only good but naturally a guide.
+                        God bless you @FaithAdekogbe.
+                        Thanks shecodeafrica & all the organizers. I'll forever make you guys proud. 🥰
                     </p>
-                    <p class="__shecodetestimp1">Chika Obi<br><span>Web Developer</span></p><br>
+                    <p class="__shecodetestimp1">Onyekachi Peace<br></p><br>
                 </div>
-                <img src="/images/hadijat.jpg" class="__shecodetestimimg">
+                <img src="/images/PeaceAmara.jpg" class="__shecodetestimimg">
             </div>
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
-                <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                <div class="__shecodetestimcard last-child">
+                    <p class="__shecodetestimp">I met lots of people that has been helping in my journey so far,also
+                        privilledge to be part of the COHORT 1 and I've
+                        learnt so much within a short period.
```

Comment:
This testimonial needs some correction:

I met lots of people that **have** been **helpful** in my journey so far. also **privileged** to be part of the COHORT 1 **of SCA's mentorship program** and I've learnt so much within a short period.

Reply:
Okay, noted